templates/todo/api/js/src/config/index.ts

```diff
@@ -68,4 +81,37 @@ const populateEnvironmentFromKeyVault = async () => {
         logger.error(`Error authenticating with Azure KeyVault.  Ensure your managed identity or service principal has GET/LIST permissions. Error: ${err}`);
         throw err;
     }
+};
+
+const populateEnvironmentFromAppConfig = async () => {
+    // If Azure AppConfig endpoint is defined
+    // 1. Login with Default credential (managed identity or service principal)
+    // 2. Overlay App Config configurations on top of ENV vars (Read from KeyVault secret if is KeyVault references)
+    const endpoint = process.env.AZURE_APPCONFIGURATION_ENDPOINT || "";
+
+    if (!endpoint) {
+        logger.warn("AZURE_APPCONFIGURATION_ENDPOINT has not been set. Configuration will be loaded from current environment.");
+        return;
+    }
+
+    try {
+        logger.info("Populating environment from Azure AppConfiguration...");
+        const credential = new DefaultAzureCredential();
+
+        const settings = await load(endpoint, credential, {
+            keyVaultOptions: {
+                // Access keyvault using the same identity, make sure the permission is set correctly.
+                credential: credential
+            }
+        });
+
+        for (const [key, value] of settings) {
+            logger.info(`Setting read from app config ${key}=${value}`);
```

Comment:
pls not log config key value, especially value

Reply:
Fixed removing log